static/lang/re-en.json

```diff
@@ -2432,6 +2432,7 @@
                 "SkillFeat": "Select a skill feat.",
                 "SubconsciousMind": "Select a subconscious mind.",
                 "ThirdImplement": "Select a third implement.",
+                "Tradition": "Select a tradition.",
```

Comment:
You ask to select a tradition but the actual choices are skills.

Reply:
Good catch!